.github/workflows/test-slow-imports-rule.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow checks that our slow-imports rule works correctly.
+# This is done by intentionally breaking the rule, running the linter, and checking
+# that we got a non-zero exit code. It's not a completely fault-proof strategy,
+# but it's an useful smoke test.
+name: Slow imports rule check
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch:
+  push:
+    paths:
+      - ".github/workflows/test-slow-imports-rule.yml"
+
+defaults:
+  run:
+    working-directory: packages/hardhat-core
+
+jobs:
+  test-slow-imports-rule:
+    name: Check that the slow-imports rule works correctly
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - uses: actions/checkout@v2
+      - name: Install
+        run: yarn
+      - name: Add slow import
+        run: echo 'import "lodash";' >> src/internal/constants.ts
+      - name: Build
```

Comment:
Not that it does something wrong, but I don't think you need this step.

Reply:
You do because several of the rules we use from `typescript-eslint` need [type information](https://typescript-eslint.io/linting/typed-linting).